Makefile.am

```diff
@@ -740,10 +741,13 @@ EXTRA_DIST += \
 	man/glib-event-glue.c
 
 # ------------------------------------------------------------------------------
+rootlibexec_LTLIBRARIES += \
+	libshared.la
```

Comment:
> i think we should give this a more specific name, i.e. "libsystemd-shared" or so...
Will do

> hmm, we stick this in rootlibexec, i.e. /usr/lib/systemd. Is ldconfig OK with that, how does that work?
libtool automatically sets the RUNPATH for all the binaries linking to it:

```
objdump -p install-tree/usr/bin/* | grep RUNPATH | uniq
  RUNPATH              /lib/systemd
```

> shouldn't this library name include the systemd version string? otherwise it might happen that during RPM/dpkg updates processes are started that load the wrong version of this library, which will break given that we won't maintain compat on this.

This is a good idea. I will add the version info as ${version}:0:0 or so.